requirements.txt

```diff
@@ -9,3 +9,4 @@ s3fs
 ipfsspec
 requests
 intake-xarray>=0.3.2
+git+https://github.com/fsspec/swiftspec.git@main#egg=swiftspec
```

Comment:
When accessing the data via `https://` protocol, there would be no need anymore for `swiftspec`.
```suggestion
```